src/ploomber/spec/dagspec.py

```diff
@@ -322,6 +322,12 @@ def _init(self, data, env, lazy_import, reload, parent_path,
 
         # there is no env.yaml
         else:
+            # wrong extension of env.yml or env.{name}.yml found
```

Comment:
Working on it

Reply:
@edublancas correct me if I'm wrong but we want to support that exact functionality for pipeline.yml right?

Reply:
throwing an error if there is a pipeline.yml? yeah, we want. but this PR is about env.yml, I think let's do it separately since the fix involves changes in different places

Reply:
I've opened an issue on the pipeline.yaml, let's keep it isolated to the env.yaml